processor/src/test/java/org/mapstruct/ap/testutil/compilation/model/CompilationOutcomeDescriptor.java

```diff
@@ -25,21 +28,37 @@
  */
 public class CompilationOutcomeDescriptor {
 
+    private static final String LINE_SEPARATOR = System.getProperty( "line.separator" );
```

Comment:
```suggestion
    private static final String LINE_SEPARATOR = System.lineSeparator();
```

Reply:
hey.. did not know that was there.. I'll fix that one